### PR TITLE
Removes http-server

### PR DESCRIPTION
**Pull request by klebba**
Addresses #158

### CODE REVIEW

File: server.js
Comment (klebba) [inline server.js:1]: These 100 lines eliminate the need for a vast trove of dependencies

Comment (theengineear) [reply server.js:1]: Ah awesome — love this change.

**Review by theengineear — APPROVED** [review]
Thanks for slimming down the dev deps here 🙏

**Comment by klebba** [comment]
Todo -- .md mime type

**Comment by klebba** [comment]
Rebased

**Review by theengineear — APPROVED** [review]
LGTM!

**Pull request merged**